project/src/main/scala/org/broadinstitute/clio/integrationtest/ClioIntegrationTestRunner.scala

```diff
@@ -76,6 +78,35 @@ class ClioIntegrationTestRunner(testClassesDirectory: File,
     *
     * @return The exit code of the integration test.
     */
+  private def tryDockerComposeJenkinsTest(): Try[Int] = Try {
+    runCommandAsync(
```

Comment:
Thanks for the in-person explanation. Can you add a comment explaining why we have to do it this way?